.github/workflows/build_unittest.yml

```diff
@@ -77,7 +77,7 @@ jobs:
           go install github.com/golang/protobuf/protoc-gen-go@v1.3.2
           sudo cp $GOPATH/bin/protoc-gen-go /usr/bin
           sudo cp $GOPATH/bin/protoc-gen-go-grpc /usr/bin
-          protos=("account" "aggregator" "auth" "chassis" "events" "fabrics" "managers" "role" "session" "systems" "task" "telemetry" "update" "compositionservice")
+          protos=("account" "aggregator" "auth" "chassis" "events" "fabrics" "managers" "role" "session" "systems" "task" "telemetry" "update" "compositionservice" "license")
           for str in ${protos[@]}; do
```

Comment:
IMHO it should be `licenses` instead of `license` @amar-shalgar  @shivacharanms  please suggest